[15.0][MIG] hr_timesheet_report_hide_responsible

adriresu opened the pull request.

ChristianSantamaria committed:
[ADD] hr_timesheet_report_hide_responsible

dalonsod committed:
[FIX] hr_timesheet_report_hide_responsible: wrong i18n files

adriresu committed:
[MIG] hr_timesheet_report_hide_responsible: Migration to 15.0

adriresu committed:
[UPD] hr_timesheet_report_hide_responsible: README.rst and Icon

ChristianSantamaria commented:
LGTM 👍 

Minor change:
Rename the UPD commit, removes the Contributors since you added that in the MIG

adriresu commented:
Renamed.
I think that adding the name in Contributors should not be mentioned on commits messages. Therefore, I removed It from the last commit ,besides, I didn't add It to MIG commit

dalonsod reviewed (approved):
Everything's fine!!!
